arch/arm/boot/dts/imx6ul-ts7180.dts

```diff
@@ -0,0 +1,778 @@
+		 */
+		timeout-sec = <300>;
+		reg = <0x54>;
+
+		/*
+		 * Not using uC WDT by default any longer. If you need features
+		 * provided by this, such as a real poweroff state, an external
+		 * WDT, or proper feedback from the uC on the platform's reboot
+		 * source (via tsmicroctl), then enable this driver (and disable
+		 * the CPU driver, wdog1,  unless you want both active).
+		 */
+
+		status = "disabled";
+	};
+
+	m41t00s: rtc@68 {
+		compatible = "st,m41t00";
+		reg = <0x68>;
+	};
+
+	/*
+	 * Reference: Documentation/devicetree/bindings/iio/imu/st_lsm6dsx.txt
+	 *
+	 *   - GYRO_INT enters CSI_HSYNC (pad F3)
+	 */
+	ism330: gyro@6a {
+		compatible = "st,ism330dlc";
+		reg = <0x6a>;
+		interrupt-parent = <&gpio4>;
+		interrupts = <20 IRQ_TYPE_LEVEL_HIGH>;    /* See MX6UL_PAD_CSI_HSYNC__GPIO4_IO20 */
+	};
+};
+
+&i2c3 {
+	clock-frequency = <400000>;
+	pinctrl-names = "default", "gpio";
+	pinctrl-0 = <&pinctrl_i2c3>;
+	pinctrl-1 = <&pinctrl_i2c3_gpio>;
+	status = "okay";
+	scl-gpios = <&gpio3 6 (GPIO_ACTIVE_HIGH | GPIO_OPEN_DRAIN)>;
+	sda-gpios = <&gpio3 5 (GPIO_ACTIVE_HIGH | GPIO_OPEN_DRAIN)>;
+
+	gpio6: gpio@28 {
+		compatible = "technologic,ts4900-gpio";
+		reg = <0x28>;
+		#gpio-cells = <2>;
+		gpio-controller;
+		base = <160>;
+		ngpios = <64>;
+	        gpio-line-names = "WIFI_RESET#",       "EN_WIFI_PWR",       "",                  "",                  
+		                  "",                  "FRAM_WP#",          "EN_CL_1",           "EN_CL_2",           
+		                  "EN_CL_3",           "EN_CL_4",           "EN_ADC1_10V",       "EN_ADC2_10V",       
+		                  "EN_ADC3_10V",       "EN_ADC4_10V",       "EN_SD_POWER",       "EN_USB_HOST_5V",    
+		                  "EN_OFF_BD_5V",      "EN_CELL_MODEM_PWR", "EN_NIMBEL_3.3V",    "EN_GPS_PWR#",       
+		                  "EN_CAN_XVR#",       "EN_232_XVR",        "EN_LS_OUT_1",       "EN_LS_OUT_2",       
+		                  "EN_LS_OUT_3",       "EN_LS_OUT_4",       "EN_LS_OUT_5",       "EN_LS_OUT_6",       
+		                  "EN_LS_OUT_7",       "MT_RESET#",         "",                  "",                  
+		                  "DIG_IN_1",          "DIG_IN_2",          "DIG_IN_3",          "DIG_IN_4",          
+		                  "SD_BOOT_JMP#",      "DIO_1_IN",          "DIO_2_IN",          "DIO_3_IN",          
+		                  "DIO_4_IN",          "DIO_5_IN",          "DIO_6_IN",          "DIO_7_IN",          
+		                  "",                  "",                  "",                  "",                
+		                  "",                  "",                  "",                  "",                  
+		                  "",                  "",                  "",                  "",                  
+		                  "",                  "",                  "",                  "",                  
+		                  "",                  "",                  "",                  "";                  
+	};
+};
+
+&iomuxc {
```

Comment:
For uniformity sake, try to have the IOMUX values tab aligned through the whole pinmux section. It looks like below some have tabs, spaces, etc. Just so long as its something uniform in the file that's fine.